Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -193,6 +193,10 @@ extension Generator {
             return try makeObject(name: name, info: info, details: details, context: context)
         case .array(let info, let details):
             return try makeTypealiasArray(name: name, info: info, details: details, context: context)
+        case .all(let schemas, _) where schemas.count == 1,
```

Comment:
Adding some comment that describes why we do like this might help others in the future...?